modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -3,7 +3,13 @@
 
 #ifdef HAVE_OPENCV_GAPI
 
+// NB: Python wrapper replaces :: with _ for classes
 using gapi_GKernelPackage = cv::gapi::GKernelPackage;
+using gapi_wip_IStreamSource_Ptr = cv::Ptr<cv::gapi::wip::IStreamSource>;
+
+// NB: Python wrapper generate code without namespace std,
+// so it cause error: "string wasn't declared"
+using std::string;
```

Comment:
Please try to use `make_capture_src(const Sring& path)`

Reply:
Add FIXME